Formula/c/caseconvert.rb

```diff
@@ -0,0 +1,19 @@
+class Caseconvert < Formula
+  desc "a tool to convert strings between different cases"
+  homepage "https://github.com/Seann-Moser/CaseConvert"
+  url "https://github.com/Seann-Moser/CaseConvert/archive/refs/tags/v1.0.1.tar.gz"
+  sha256 "abaafd92a5853f50ddc3e4018f6b5160eebc661df7f1eabc1276ff26649b6c7e"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "mod", "tidy"
+    system "go", "mod", "vendor"
```

Comment:
Go mod tidying and vendoring is not needed to build a binary.

Reply:
Updated and removed go mod vendor and tidy